beaker-vue/src/components/cell/NextBeakerQueryCell.vue

```diff
@@ -130,7 +130,7 @@ const isQueryActive = computed(() => {
     if (mockStickyForce.value) return true; // Force active for testing
 
     const queryStatus = cell.value.metadata?.query_status;
-    return queryStatus === 'in-progress';
+    return queryStatus === 'in-progress' && ['busy', 'awaiting_input'].includes(cell.value.status);
```

Comment:
Removes previously "active" query cell stickyness if query cell errors and goes to `idle` even though it was last seen as in-progress (and not marked as `success`)